saga/saga_state_test.go

```diff
@@ -55,26 +55,3 @@ func TestSagaState_Copy(t *testing.T) {
 		t.Error(fmt.Sprintf("Copy Should Preserve SagaId"))
 	}
 }
-
-func TestSagaState_SagaStateNotMutatedDuringUpdate(t *testing.T) {
```

Comment:
The tests in saga_test.go already check for this (and were very useful in diagnosing what was going on).